### DISCUSSION TITLE
User-Groups and League-Groups Profiles

### DISCUSSION THREAD

**Discussion by ghost** [RetroAchievements/RAWeb#787] [Ideas]
It would be nice in the future (v2 etc.) to create User-Group and allow multipe users join a Group, collect Achievements on same shared profile while earning it on their personal profile as well. And League-Group Profiles for Competitive Events, public and private.

**User Groups:**
The idea behind User-Groups is to have a shared profile for the unlocks/earnings without switching accounts. Duplicate unlocks/earnings will be ignored so only 1 unlock per Group.

```
User-Group

User_1 has unlocked Achievement_A
User_1 has unlocked Achievement_B
User_2 has unlocked Achievement_A <- this will be ignored due to being a duplicate
User_2 has unlocked Achievement_C

This Group has earned 3 Achievements.
```

This would allow to create many shared profiles (Groups) for Team vs Team Events or if friends want to go after Achievements with one shared profile instead of sharing a original user profile (being logged in same time etc).

Basically User-Group would function as one Unit.

**League-Groups:**
This would function in similar way to User-Group with the difference that it doesn't work as one Unit.

This could be used for all kind of Community Events. Each League gets its own Leaderboard entries and unlocks (fresh start without touching each users personal profile) so users already unlocked earnings don't count here. Each League/Event will start from 0.

Also:
- Allow users to join a Group/League via Invite
- Allow Group/League creators to remove users from their Group/League
